app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = [Person(human["name"], human["age"]) for human in people]
+
+    for human in people:
+        if "wife" in human and human["wife"] is not None:
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif "husband" in human and \
+                human["husband"] is not None:
+            Person.people[human["name"]].husband \
```

Comment:
It is better to create a new variable that will contain `Person.people[human["name"]]`  and delete `\`